esp-hal/src/clock/mod.rs

```diff
@@ -274,7 +274,6 @@ pub struct Clocks<'d> {
     pub pll_48m_clock: HertzU32,
     #[cfg(esp32h2)]
     pub pll_96m_clock: HertzU32,
-    // TODO chip specific additional ones as needed
```

Comment:
these comments serve no purpose anymore - we know we need to have changes here when adding new targets